src/Api/Serializer/RestSerializer.php

```diff
@@ -36,21 +40,38 @@ public function __construct(Service $api, $endpoint)
     }
 
     /**
-     * @param CommandInterface $command Command to serialized
+     * @param CommandInterface $command Command to serialize
```

Comment:
maybe change this to "Command to serialize into a request" to make it clearer what this is doing